5_analyze_data/notebooks/UMAP_analysis/UMAPutils.py

```diff
@@ -0,0 +1,68 @@
+import pathlib
+import pandas as pd
+
+def split_data(pycytominer_output: pd.DataFrame):
+    """
+    split pycytominer output to metadata dataframe and np array of feature values
+    Parameters
+    ----------
+    pycytominer_output : pd.DataFrame
+        dataframe with pycytominer output
+    Returns
+    -------
+    pd.Dataframe, np.ndarray
+        metadata dataframe, feature values
```

Comment:
Very minor suggestion: consider keeping your docstring spacing consistent as per the NumPy docstring style in this file.
```suggestion
    split pycytominer output to metadata dataframe and np array of feature values
    
    Parameters
    ----------
    pycytominer_output : pd.DataFrame
        dataframe with pycytominer output
    
    Returns
    -------
    pd.Dataframe, np.ndarray
        metadata dataframe, feature values
```

Reply:
👍